### DISCUSSION TITLE
内置表达式作用域是否能获取依赖的实例？

### DISCUSSION THREAD

**Discussion by wweggplant** [alibaba/formily#3283] [Q&A 2.x] [answered]
$deps和$dependencies都是获取依赖字段的值，想要获取实例的话，需要通过query来查找。如果有依赖字段的实例的话，比如我想获取依赖字段的DataSource中的某个属性，这种还是很方便的。

**Comment by janryWang** [comment, marked as answer]
dependencies:['aa#dataSource']

**Reply by wweggplant** [reply]
感谢
